rails/benchmarks/bm_scaffold_create.rb

```diff
@@ -4,13 +4,6 @@
 require 'active_record'
 require_relative 'support/benchmark_rails'
 
-ENV['RAILS_ENV'] = 'production'
-
-class NullLogger < Logger
-  def initialize(*args);end
-  def add(*args, &block);end
-end
```

Comment:
@SamSaffron @kirs We have to log because logging do affect performance quite a bit. Setting to a NullLogger was not running through any of the logging code.